tests/api.c

```diff
@@ -6638,6 +6660,95 @@ static void test_wolfSSL_X509_LOOKUP_load_file(void)
              !defined(NO_FILESYSTEM) && !defined(NO_RSA) */
 }
 
+static void test_wolfSSL_X509_STORE_CTX_set_time(void)
+{
+    #if defined(OPENSSL_EXTRA)
+    WOLFSSL_X509_STORE_CTX*  ctx;
+    time_t ctime;
+
+    printf(testingFmt, "wolfSSL_X509_set_time()");
+    ctx = wolfSSL_X509_STORE_CTX_new();
+    ctime = 365*24*60*60;
+    wolfSSL_X509_STORE_CTX_set_time(ctx, 0, ctime);
+    AssertTrue((ctx->param->flags & WOLFSSL_VERIFY_TIME) == WOLFSSL_VERIFY_TIME);
+    AssertTrue(ctx->param->check_time == ctime);
+    wolfSSL_X509_STORE_CTX_free(ctx);
+
+    printf(resultFmt, passed);
+    #endif /* OPENSSL_EXTRA */
+}
+
+static void test_wolfSSL_CTX_add_client_CA(void)
+{
+    #if defined(OPENSSL_EXTRA) && !defined(NO_RSA)
+    WOLFSSL_CTX* ctx;
+    WOLFSSL_X509* x509;
+    WOLFSSL_X509* x509_a;
+    STACK_OF(WOLFSSLX509_NAME)* ca_list;
+    int ret = 0;
+
+    printf(testingFmt, "wolfSSL_CTX_add_client_CA()");
+    ctx = wolfSSL_CTX_new(wolfSSLv23_client_method());
+    /* Add client cert */
+    x509 = wolfSSL_X509_load_certificate_file(cliCertFile, SSL_FILETYPE_PEM);
+    AssertNotNull(x509);
+    ret = wolfSSL_CTX_add_client_CA(ctx, x509);
+    AssertIntEQ(ret ,SSL_SUCCESS);
+    ca_list = wolfSSL_SSL_CTX_get_client_CA_list(ctx);
+    AssertNotNull(ca_list);
+    /* Add another client cert */
+    x509_a = wolfSSL_X509_load_certificate_file(cliCertFile, SSL_FILETYPE_PEM);
+    AssertIntEQ(wolfSSL_CTX_add_client_CA(ctx, x509_a),SSL_SUCCESS);
+
+    wolfSSL_X509_free(x509_a);
+    wolfSSL_X509_free(x509);
+    wolfSSL_CTX_free(ctx);
+
+    printf(resultFmt, passed);
+    #endif /* OPENSSL_EXTRA && !NO_RSA */
+}
+
+static void test_wolfSSL_CTX_set_srp_password(void)
+{
+#if defined(OPENSSL_EXTRA) && defined(WOLFCRYPT_HAVE_SRP) \
+    && !defined(NO_SHA256) && !defined(RC_NO_RNG)
+    WOLFSSL_CTX* ctx;
+    const char *username = "TESTUSER";
+    const char *password = "TESTPASSWORD";
+    int r;
+
+    printf(testingFmt, "wolfSSL_CTX_set_srp_password()");
+    ctx = wolfSSL_CTX_new(wolfSSLv23_client_method());
+    AssertNotNull(ctx);
+    r = wolfSSL_CTX_set_srp_username(ctx, (char *)username);
```

Comment:
I misunderstood because internal function require calling order.
OpenSSL is not required, So I added a variable to obtain password string and change 2 function code.